### ISSUE TITLE
Transmission.app crashing while Karabiner Elements is running

### ISSUE THREAD

**Issue by TSFoster** [tekezo/Karabiner-Elements#22]
If Karabiner Elements is running, pressing a key while using Transmission causes the app to crash. I presume this might happen with other applications, too. Error message:

```
Exception Name: NSRangeException
Description: -[__NSCFConstantString characterAtIndex:]: Range or index out of bounds
User Info: (null)

0   CoreFoundation                      0x00007fffbc3277bb __exceptionPreprocess + 171
1   libobjc.A.dylib                     0x00007fffd0a86a2a objc_exception_throw + 48
2   CoreFoundation                      0x00007fffbc3a4a65 +[NSException raise:format:] + 197
3   CoreFoundation                      0x00007fffbc2a0686 -[__NSCFString characterAtIndex:] + 102
4   Transmission                        0x000000010954c180 Transmission + 131456
5   AppKit                              0x00007fffba7a37ec -[NSWindow(NSEventRouting) _reallySendEvent:isDelayedEvent:] + 4086
6   AppKit                              0x00007fffba7a242a -[NSWindow(NSEventRouting) sendEvent:] + 541
7   AppKit                              0x00007fffba643a1c -[NSApplication(NSEvent) sendEvent:] + 4768
8   AppKit                              0x00007fffb9f29009 -[NSApplication run] + 1002
9   AppKit                              0x00007fffb9ef3a8a NSApplicationMain + 1237
10  Transmission                        0x000000010952e7b3 Transmission + 10163
11  Transmission                        0x000000010952e300 Transmission + 8960
12  ???                                 0x0000000000000001 0x0 + 1
```

Karabiner Elements v0.90.25  
macOS 10.12 beta (16A313a)  
2012 15" MacBook Pro w/ Retina Display

**Comment by tekezo** [comment]
Thank you for feedback.

I believe this issue has been fixed at Karabiner-Elements 0.90.29.
Please confirm it.

**Comment by TSFoster** [comment]
Yes, this is fixed for me, thank you. Closing
